readme.txt

```diff
@@ -1,97 +1,245 @@
+* Comments — Replace your default comment form with an improved system with integrated social media login options.
+* Comment Likes — Allows readers to like other comments to show their agreement, approval, or appreciation.
+* Contact Form — Offer your readers the ability to get in touch, without giving out your personal email address.
+* Custom CSS — Customize the appearance of your theme without creating a child theme or worrying about updates overwriting your customizations.
+* Custom Content Types — Adds custom post types (CPTs) to your site.
+* Downtime Monitor — Alerts you via electronic mail if your site goes down to ensure you keep uptime.
+* Enhanced Distribution — Allows your content to be included in the WordPress.com “firehose” of public blog content.
+* Extra Sidebar Widgets — Extra widgets you can add to your blog, including RSS Links, Twitter Timelines, and Facebook Like Boxes.
+* Gravatar Hovercards — Make your Gravatar profile visible to those viewing your blog.
+* Google Analytics (GA) — Track your WordPress site statistics thanks to Google Analytics.
+* Infinite Scroll — Pulls the next posts automatically into view when the reader approaches the bottom of the page.
+* JSON API — Authorizes applications and services to securely connect to your blog, and allows them to use your content or offer you new functionality.
+* Lazy Load Images — Makes pages load faster by only lazy loading images that are on the screen, and loads other images as the user scrolls
+* Likes — Allows readers to show their appreciation for your posts with a single click.
+* Markdown — Allows you to compose posts and comments with links, lists, and other styles using regular characters and punctuation marks. Markdown is used by writers and bloggers who want a quick and easy way to write rich text without having to take their hands off the keyboard.
+* Notifications — Receive notifications for new comments and Likes in your admin bar and on your mobile device.
+* oEmbed Support — easily embed images, posts, and links from Facebook and Instagram.
+* Pinterest Block — Allows you to embed boards, profiles, and pins to your site.
+* Plugin Management — Allows easy site maintenance by choosing which plugins update automatically.
+* Post by Email — Publish posts using any mail client.
+* Protect — Protect your site from traditional and distributed brute force login attacks.
+* Publicize — Share new posts on social media networks automatically, or schedule future shares with custom messages.
+* Recurring Payments — Add a payment button specifically for a recurring donation, using Stripe as the payment gateway.
+* Related Posts  — Show contextual posts your visitors might be interested in reading after they’re done with their current post.
+* Secure Auth —  Secure WordPress.com powered login used by millions of sites with optional 2FA (two factor authentication) for extra protection.

```

Comment:
Is this the same as SSO, featured a few lines below?

Reply:
Yes, I'll delete SSO